scripts/api_dump_generator.py

```diff
@@ -2370,8 +2370,10 @@ def __init__(self, rootNode, constants, aliases, parentName):
         self.lengthMember = False
         lengthString = rootNode.get('len')
         if lengthString is not None:
-            if "ename:" in lengthString:
+            if "ename:" in lengthString or "latexmath:" in lengthString:
                 lengthString = rootNode.get('altlen')
+                if (lengthString == "(rasterizationSamples + 31) / 32"):
+                    lengthString = "rasterizationSamples / 32 + (31 / 32)"
```

Comment:
On second thought, I don't think this works for integer math

Reply:
rasterizationSamples + 31 >> 5

Reply:
Whoops, thats right. Integer math doesn't do fractions. The bitshift indeed does work here, even if its even less clean. 
Would be excellent if there was a comment detailing why this is special cased and why a bitshift by 5 is needed.